Added documentation for multiple battery monitoring (#2167)
zmkfirmware/zmk: docs/docs/features/battery.md

If your keyboard has a battery sensor, ZMK will report its battery level to the connected bluetooth host and show it on the keyboard's display, if it has one.

> For split keyboards, only the battery level of the central (usually left) side is reported over bluetooth. You can configure ZMK to report the battery levels of both sides, but there's no guarantee that the OS will natively read both battery values.

Author: Meh, it's not like virtually no host will display both values: from the original pull request #2045, it appears that macOS natively shows both battery values, which if true would make up for a big percentage of users. 
Should we still write "virtually all hosts will not display this information"?
By the way, which "ZMK default widgets" are you referring to?

> For split keyboards, only the battery level of the central (usually left) side is reported over bluetooth. You can configure ZMK to report the battery levels of both sides, but there's no guarantee that the OS will natively read both battery values.

Author: (I've modified the section as suggested but instead of linkind to ../config/system, I linked to ../config/battery

> For split keyboards, only the battery level of the central (usually left) side is reported over bluetooth. You can configure ZMK to report the battery levels of both sides, but there's no guarantee that the OS will natively read both battery values.

Author: Ohhh
Nope, sorry 😅
Someone else would have to check that.
I found [this post](https://www.reddit.com/r/CustomKeyboards/comments/koc9hw/finally_fully_wireless_soflezmknicenano/) which shows two different battery levels on the display, not sure if it's relevant

> For split keyboards, only the battery level of the central (usually left) side is reported over bluetooth. You can configure ZMK to report the battery levels of both sides, but there's no guarantee that the OS will natively read both battery values.

Reviewer: It requires custom code to do that, the default status screens don't support that indeed.

> For split keyboards, only the battery level of the central (usually left) side is reported over bluetooth. You can configure ZMK to report the battery levels of both sides, but there's no guarantee that the OS will natively read both battery values.

Author: > > Meh, it's not like virtually no host will display both values: from the original pull request #2045, it appears that macOS natively shows both battery values, which if true would make up for a big percentage of users. Should we still write "virtually all hosts will not display this information"?
> 
> I missed this; in the pull request before Pete took over (#1243) Katona had to write an application for macOS to display this information, and I assumed that was still the status quo. "many hosts", then.
> 
> > By the way, which "ZMK default widgets" are you referring to?
> 
> ZMK's built-in display widgets. The ones used by default if you have a display, such as an OLED, on your keyboard. (...It sounds like you don't have one.)

done

> For split keyboards, only the battery level of the central (usually left) side is reported over bluetooth by default. ZMK can be [configured to report the battery levels for peripherals](../config/battery#peripheral-battery-monitoring), but virtually all hosts will not display this information without additional configuration or the use of third party utilities.

Reviewer: ```suggestion
For split keyboards, only the battery level of the central (usually left) side is reported over bluetooth by default. ZMK can be [configured to report the battery levels for peripherals](../config/battery.md#peripheral-battery-monitoring), but virtually no host system will display this information without additional configuration or the use of third party utilities.
```
